### PR TITLE
Add image version to CLI (fixes #105)

### LINKED ISSUE #105: treehouses image
returns /boot/version.txt content

### PR DESCRIPTION
**Pull request by rhuturajm**

### CODE REVIEW

**Comment by i5o** [comment]
Hey @rhuturajm.
I have just tested this and while the file (`/boot/version` -> `/boot/version.txt`) is wrong, the code itself is not giving the output we want.
```
pi@raspberrypi:~ $ image=$(cat /boot/version.txt)
pi@raspberrypi:~ $ echo $((image))
-58
pi@raspberrypi:~ $ 
```

so, how do we fix this?

- [ ] rename /boot/version to /boot/version.txt
- [ ] checkout if `cat` could do this in oneline 

let me know if you have any question

**Comment by rhuturajm** [comment]
@i5o Hey can you tell how can i simulate the function in cli?

**Comment by i5o** [comment]
@rhuturajm 
image

**Comment by rhuturajm** [comment]
Thank you

**Comment by i5o** [comment]
Hey @rhuturajm , good work so far.
However `cat` prints to stdout, so there is no need for echo.

So you may want to check shellcheck.net to see what's wrong
https://www.shellcheck.net/
image

**Comment by rhuturajm** [comment]
yes working on that

File: cli.sh [hunk 1]
```diff
@@ -67,6 +67,17 @@ function help {
       echo "      Prints the version of $(basename "$0") currently installed."
       echo ""
       ;;
+    imageversion)
+      echo ""
+      echo "Usage: $(basename "$0") imageversion"
+      echo ""
+      echo "Returns the version of $(basename "$0") image which is currently running on RPI"
```
Comment (i5o) [inline cli.sh]: Returns the version of the system image which is currently running

File: cli.sh [hunk 2]
```diff
@@ -67,6 +67,17 @@ function help {
...
+      echo ""
+      echo "Example:"
+      echo "  $(basename "$0") imageversion"
+      echo "      Prints the version of $(basename "$0") image currently installed."
```
Comment (i5o) [inline cli.sh]: Prints the current version of the system image

File: cli.sh [hunk 3]
```diff
@@ -288,6 +299,7 @@ function help {
       echo "   password <password>                      changes the password for 'pi' user"
       echo "   sshkeyadd <public_key>                   adds a public key to 'pi' and 'root' user's authorized_keys"
       echo "   version                                  returns the version of $(basename "$0") command"
+      echo "   imageversion                             returns version of the treehouses image installed"
```
Comment (i5o) [inline cli.sh]: returns the version of the system installed

**Pull request merged**